.ci/targets/build_all_plugins.yaml

```diff
@@ -5,3 +5,4 @@ tasks:
     script: .ci/scripts/create_all_plugins_app.sh
   - name: build all_plugins
     script: .ci/scripts/build_all_plugins.sh
+    args: ["windows"]
```

Comment:
Didn't find an easy way to inject Env. variable to yaml. It would be nice to consolidate different platforms when recipes has a good support on yaml Env. parser. For now, I am using separate .yaml targets for different platforms.